scania-digital-design-system/sdds
# feat(table): multiselect

**Describe pull-request**  
Creating Multiselect feature for data table component

**Solving issue**  
Fixes: AB#1697

**How to test**
1. Go to Storybook
2. Check Data Table component -> Web Component
3. Play around multi-select feature (select one, deselect, select all, deselect one)
